lib/linguist/languages.yml

```diff
@@ -3331,6 +3331,18 @@ Jupyter Notebook:
   aliases:
   - IPython Notebook
   language_id: 185
+Just:
+  type: programming
+  aliases:
+  - Justfile
+  color: "#384d54"
+  tm_scope: source.just
+  filenames:
+  - JUSTFILE
+  - Justfile
+  - justfile
```

Comment:
@casey:  [This](https://github.com/casey/just#hiding-justfiles) talks about `.justfile` naming. Would it not make sense to also include it in this list?

Reply:
Note that in the wild, users also frequently use `.just` as a file suffix for Justfiles.

Reply:
Neither of these extensions are sufficiently popular on GitHub for inclusion right now. The `.just` extension was discussed earlier in this PR [here](https://github.com/github/linguist/pull/6167#pullrequestreview-1185649435).

Reply:
I understood the popularity requirement as something to be fulfilled for new languages.
The PR is not producing support for a new language (the language is already known to Linguist), but instead adding file aliases to an already known language.

Reply:
The popularity requirement applies for all new extensions and filenames, regardless of whether it's a new or known language. See the details in CONTRIBUTING.md